honeybee_vtk/file_writers.py

```diff
@@ -0,0 +1,229 @@
+                       target_folder)
+
+    # Names of VTK files to be written
+    file_names = list(grouped_points.keys())
+
+    # write grid points to a vtk file if grids are found in HBJSON
+    grids = check_grid(hbjson)
+
+    # If grids are there in HBJSON and they are requested
+    if grids and include_grids:
+        grid_file_names = _write_grids(grids, vtk_writer, vtk_extension, target_folder)
+        file_names.extend(grid_file_names)
+
+    # Write vectors if they are requested
+    if include_vectors:
+        to_write_vectors = [hb_types, grouped_points, grids, include_grids, vtk_writer,
+                            vtk_extension, target_folder]
+        vector_file_names = _write_vectors(*to_write_vectors)
+        file_names.extend(vector_file_names)
+
+    # Use the name of HBJSON if file name is not provided
+    if not file_name:
+        name = '.'.join(os.path.basename(file_path).split('.')[:-1])
+        file_name = clean_string(name)
+
+    # remove extension if provided by user
+    file_name = file_name if not file_name.lower().endswith('.zip') else file_name[:-4]
+    
+    # Create a .zip file to capture all the generated .vtk files
+    zip_file = os.path.join(target_folder, file_name + '.zip')
+    zipobj = ZipFile(zip_file, 'w')
+
+    # Capture vtk files in a zip file.
+    os.chdir(target_folder)
+    for file_name in file_names:
+        zipobj.write(file_name + vtk_extension)
+    zipobj.close()
+
+    # Delete vtk files if there is permission
+    for file_name in file_names:
+        try:
+            file_name = os.path.join(target_folder, file_name + vtk_extension)
+            os.remove(file_name)
+        except OSError:
+            warnings.warn(
+                f'Honeybee does not have permission to delete {file_names}.'
+                ' You may please delete the .vtk files manually.'
+            )
+    # Return the path where the .zip file is written
+    return zip_file
+
+
+def write_html(file_path, file_name, target_folder, include_grids, include_vectors,
+               hbjson):
+    """Write an HTML file with VTK objects embedded into it.
+
+    Args:
+        file_path: A text string for a valid path to the HBJSON file.
+        file_name: A text string for the name of the .zip file to be written. If no
+            text string is provided, the name of the HBJSON file will be used as a
+            file name for the .zip file.
+        target_folder: A text string to a folder to write the output file. The file
+            will be written to the current folder if not provided.
+        include_grids: A boolean. Defaults to True. Grids will not be extracted from
+            HBJSON if set to False.
+        include_vectors: A boolean. Defaults to True. Vector arrows will not be created
+            if set to False.
+        vtk_writer: A VTK writer object.
+        vtk_extension: A text string for the file extension to be used.
+        hbjson: A dictionary.
```

Comment:
It feels really strange that hbjson is the last argument in these functions. It feels that it should be the first as it is the most important argument.